site/docs/examples/wasm-to-distributed-program.md

```diff
@@ -0,0 +1,376 @@
+---
+id: wasm-to-decentralized-program
+title: Turning Existing WebAssembly Applications Into Distributed Programs
+sidebar_label: Turning Existing WebAssembly Applications Into Distributed Programs
+---
+
+**Or, how to make an eternal Game of Life Simulator.**
+
+![alt text](https://miro.medium.com/max/1510/1*AXw36HaBjYVQeuERis-U6A.png "Decentralized Game of Life built on Wavelet")
+
+One of the benefits of WebAssembly that we keep going on about is the ability to re-use existing code. 
```

Comment:
Change to "This tutorial aims to show how easy it is to re-use Rust application code in Wavelet."